src/storage/store.js

```diff
@@ -10,6 +11,34 @@ const validator = new Validator();
 import { EventTarget } from "event-target-shim";
 import { fetchRandomDefaultAvatarId, generateRandomName } from "../utils/identity.js";
 
+export const defaultMaterialQualitySetting = (function() {
+  const MATERIAL_QUALITY_OPTIONS = ["low", "medium", "high"];
+  let isMobile;
+  try {
+    // HACK: AFRAME is not available on all pages, so we catch the ReferenceError.
+    // We could move AFRAME's device utils into a separate package (or into this repo)
+    // if we wanted to use these checks without having to import all of AFRAME.
+    isMobile = AFRAME.utils.device.isMobile() || AFRAME.utils.device.isMobileVR();
```

Comment:
Instead of wrapping this in a try/catch you could just check for `window.AFRAME`.